docs/REST API/rest-api.md

```diff
@@ -1,24 +1,5 @@
 # REST API
 
-- [Authentication](#authentication)
-- [Activities](#activities)
-- [Fleet configuration](#fleet-configuration)
-- [File carving](#file-carving)
-- [Hosts](#hosts)
-- [Labels](#labels)
-- [Mobile device management (MDM)](#mobile-device-management-mdm)
-- [Policies](#policies)
-- [Queries](#queries)
-- [Schedule (deprecated)](#schedule)
-- [Scripts](#scripts)
-- [Sessions](#sessions)
-- [Software](#software)
-- [Targets](#targets)
-- [Teams](#teams)
-- [Translator](#translator)
-- [Users](#users)
-- [API errors](#api-responses)
-
```

Comment:
These links don't actually show up on the website, so it doesn't hurt to keep them:
<img width="1511" alt="Screenshot 2023-09-08 at 4 07 21 PM" src="https://github.com/fleetdm/fleet/assets/3065949/c813daaa-09c4-41ff-89de-3be68a5f7c44">

I would guess they're there for anyone consuming the API docs via markdown files on GitHub (probably not very many people, but users on an older version of Fleet would need to reference the API this way, since we always have the latest on the website). @eashaw or @mike-j-thomas  do you happen to know if these links were left in the file intentionally?

Reply:
>do you happen to know if these links were left in the file intentionally?

@rachaelshaw Yes, we keep those links for users looking at the docs in the GitHub repo.

Reply:
> we keep those links for users looking at the docs in the GitHub repo.

> probably not very many people, but users on an older version of Fleet would need to reference the API this way

@rachaelshaw do we need to support these users when it's adding more surface area to the docs? I don't think so but up to you.